libr/core/cmd_print.c

```diff
@@ -5330,7 +5330,18 @@ static int cmd_print(void *data, const char *input) {
 				goto beach;
 			}
 		}
-		
+
+		if (input[1] == 'x') { // pdx
+			if (input[2] == '?') {
+				r_cons_printf ("Usage: pdx [hexpairs] - disassemble hexpairs\n");
+				return 0;
+			} else {
+				r_core_cmdf (core, "pd @x:%s", r_str_trim_ro (input + 2));
```

Comment:
probably better to parse the hexpair string and return it back into hexpairs from the raw buffer. this way it wont be possible to inject code